File: python/7 kyu/307.py
# https://www.codewars.com/kata/583df40bf30065fa9900010c/train/python

# Passed

def get_mean(arr,x,y):
    if not all(limit > 1 and limit <= len(arr) for limit in [x, y]):
        result = -1
    else:
        first_mean = sum(arr[:x]) / x
        second_mean = sum(arr[-y:]) / y
        result = (first_mean + second_mean)/ 2
        
    return result

output = get_mean([1, 3, 2, 4], 2, 3)
print(output)

File: nodejs/7 kyu/307.js
// https://www.codewars.com/kata/556196a6091a7e7f58000018/train/javascript

// Passed

function largestPairSum(numbers) {
  let sortedArr = numbers.sort((a, b) => b - a);
  let largestPairSum = sortedArr[0] + sortedArr[1];
  return largestPairSum;
}

const output = largestPairSum([10, 14, 2, 23, 19]);
console.log(output);